docs/content/en/docs/Architecture/container.md

```diff
@@ -45,28 +45,28 @@ The Image support matrix in [here](/docs/reference/image_matrix) lists all the c
 To inspect an image and run it locally, you can use a container engine like Docker or Podman:
 
 ```
-$ docker pull quay.io/kairos/core-alpine:v1.1.0
+$ docker pull {{< registryURL >}}/core-{{< flavor >}}:{{< kairosVersion >}}
 ```
 
 We can run it locally with docker as a container to inspect it, as it is runnable:
 
 ```
-$ docker run -ti --rm quay.io/kairos/core-alpine:v1.1.0
+$ docker run -ti --rm {{< registryURL >}}/core-{{< flavor >}}:{{< kairosVersion >}}
 / # cat /etc/os-release
-NAME="kairos-core-alpine"
-VERSION="v1.1.0"
-ID="kairos"
-ID_LIKE="kairos-core-alpine"
-VERSION_ID="v1.1.0"
-PRETTY_NAME="kairos-core-alpine v1.1.0"
-ANSI_COLOR="0;32"
-BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
-HOME_URL="https://github.com/kairos-io/kairos"
-IMAGE_REPO="quay.io/kairos/core-alpine"
-IMAGE_LABEL="latest"
-GITHUB_REPO="kairos-io/kairos"
-VARIANT="core"
-FLAVOR="alpine"/ #
+...
+KAIROS_NAME="kairos-core-{{< flavor >}}"
+KAIROS_VERSION="{{< kairosVersion >}}"
+KAIROS_ID="kairos"
+KAIROS_ID_LIKE="kairos-core-{{< flavor >}}"
+KAIROS_VERSION_ID="{{< kairosVersion >}}"
+KAIROS_PRETTY_NAME="kairos-core-{{< flavor >}} {{< kairosVersion >}}"
+KAIROS_BUG_REPORT_URL="https://github.com/kairos-io/kairos/issues"
+KAIROS_HOME_URL="https://github.com/kairos-io/kairos"
+KAIROS_IMAGE_REPO="{{< registryURL >}}/core-{{< flavor >}}"
+KAIROS_IMAGE_LABEL="latest"
+KAIROS_GITHUB_REPO="kairos-io/kairos"
```

Comment:
@mauromorales last one: https://github.com/kairos-io/kairos/blob/d7d9478f2320956b83f566a2f583ae388b8baf09/internal/agent/upgrade.go#L35

Reply:
oh, right, we care of that automatically. nevermind, ignore me!